ttml2srt.py

```diff
@@ -72,7 +86,7 @@ def _load_ttml_doc(self, filepath):
                 ('tickRate', 0, lambda x: int(x)),
                 ('timeBase', 'media', lambda x: x),
                 ('clockMode', '', lambda x: x),
-                ('frameRateMultiplier', 1, lambda x: int(x)),
+                ('frameRateMultiplier', 1.0, lambda x: self._parse_frame_rate_multiplier),
```

Comment:
Slight oversight here. I suspect you meant to replace the lambda with `self._parse_frame_rate_multiplier`, not return it.

Reply:
done